pelican/tools/templates/pelicanconf.py.jinja2

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*- #
+"""Main settings file."""
 
-AUTHOR = {{author}}
-SITENAME = {{sitename}}
-SITEURL = ''
+AUTHOR = "{{author}}"
+SITENAME = "{{sitename}}"
+SITEURL = ""
 
-PATH = 'content'
+PATH = "content"
 
-TIMEZONE = {{timezone}}
+TIMEZONE = "{{timezone}}"
 
-DEFAULT_LANG = {{lang}}
+DEFAULT_LANG = "{{lang}}"
```

Comment:
Are you sure about the added quotes to `AUTHOR`, `SITENAME`, `TIMEZONE` and `DEFAULT_LANG`?